test/static.test.js

```diff
@@ -21,6 +21,9 @@ const indexContent = fs.readFileSync('./test/static/index.html').toString('utf8'
 const foobarContent = fs.readFileSync('./test/static/foobar.html').toString('utf8')
 const deepContent = fs.readFileSync('./test/static/deep/path/for/test/purpose/foo.html').toString('utf8')
 const innerIndex = fs.readFileSync('./test/static/deep/path/for/test/index.html').toString('utf8')
+const allThreeBr = fs.readFileSync('./test/static-pre-compressed/all-three.html.br').toString('utf8')
+const gzipOnly = fs.readFileSync('./test/static-pre-compressed/gzip-only.html.gz').toString('utf8')
+const uncompressedStatic = fs.readFileSync('./test/static-pre-compressed/uncompressed.html').toString('utf8')
```

Comment:
I won't convert them to utf8, they should be binaries.

Reply:
Ottimitting it would still covert them to utf8. I mean to keep them as buffers.

Reply:
Ah. Without that call the tests are failing now. `request.body` is read in the tests as a string. Should I be converting the body into a buffer before checking for the match or does this point to a bug?

Reply:
I think you can use `res.rawPayload` instead. It should be a buffer. https://github.com/fastify/light-my-request/blob/bfffc360ef291df9bfbda9675bf80fc04d534dd7/lib/response.js#L135